### ISSUE TITLE
Super frustrating issue

### ISSUE THREAD

**Issue by d2fan08** [kolton/d2bot-with-kolbot#916]
So i updated my bot, replaced everything with that new download and now my hdin wont switch to his concentration aura. He just stay on tele while attacking. Any idea how to fix this or what could be going on?

**Comment by texxxmex** [comment]
Post char config.

**Comment by thatguy692** [comment]
try this for your attacks setup

> Config.AttackSkill[0] = -1; // Preattack skill. Not implemented yet.
> Config.AttackSkill[1] = 112; // Primary skill to bosses.
> Config.AttackSkill[2] = 113; // Primary aura to bosses
> Config.AttackSkill[3] = 112; // Primary skill to others.
> Config.AttackSkill[4] = 113; // Primary aura to others.
> Config.AttackSkill[5] = 101; // Secondary skill if monster is immune to primary.
> Config.AttackSkill[6] = 124; // Secondary aura.

should be working just fine :+1:

**Comment by d2fan08** [comment]
Thanks man, working right now but ill give it a shot when i get home

**Comment by d2fan08** [comment]
Hey, i tried switch the skills like you have them and it still acting up... any other ideas?

**Comment by d2fan08** [comment]
anyone have any other ideas??

**Comment by crush-you** [comment]
are you sure you have all the right skills selected? Your first version had blessed hammer as the attack with Conviction as the aura. The setup that thatguy692 posted was with BH as attack, Concentration as the aura, and Holy Bolt + Redemption as the secondary setup for immunes. If you are missing some of those skills it might give you a problem.

**Comment by d2fan08** [comment]
yeah man, i did it the way he had posted and the way that i did beforehand. Like I said it was working fine and then one day just wouldn't switch skills. I still haven't figured it out if anyone has any suggestions.

**Comment by crush-you** [comment]
I would look at updating your bot folder - I just copied your config file, changed the skills to thatguy692's setup, and it works fine with my pally. He does no damage because he's normally FoH but all the skills are switching and working properly.

**Comment by d2fan08** [comment]
do you know how to go about updating my bot folder?

**Comment by crush-you** [comment]
normally you would right click on the folder and click "SVN update". This time I'm not as sure, I saw Noah say somewhere that there were some significant changes lately that may require something more extensive, like a full download. I haven't updated mine in a while so I can't be positive about that.

**Comment by d2fan08** [comment]
well i appreciate the help,still no luck on my end though. Let me know if you guys have any more ideas!

**Comment by d2fan08** [comment]
still not working guys
